x-pack/plugins/observability_ai_assistant/public/hooks/use_observability_ai_assistant_router.ts

```diff
@@ -47,39 +47,16 @@ export function useObservabilityAIAssistantRouter(): StatefulObservabilityAIAssi
       ...observabilityAIAssistantRouter,
       push: (...args) => {
         const next = link(...args);
-
         history.push(next);
       },
-      navigateToConversationsApp: (path, ...args) => {
-        const [_, route, routeParam] = path.split('/');
-
-        const sanitized = routeParam.replace('{', '').replace('}', '');
-
-        const pathKey = args[0]?.path;
-
-        if (typeof pathKey !== 'object') {
-          return;
-        }
-
-        if (Object.keys(pathKey).length === 0) {
-          navigateToApp('observabilityAIAssistant', {
-            path: route,
-          });
-          return;
-        }
-
-        if (Object.keys(pathKey).length === 1) {
-          navigateToApp('observabilityAIAssistant', {
-            // @ts-expect-error
-            path: `${route}/${pathKey[sanitized]}`,
-          });
-          return;
-        }
-      },
       replace: (path, ...args) => {
         const next = link(path, ...args);
         history.replace(next);
       },
+      navigateToConversationsApp: (path, ...args) => {
```

Comment:
curious why we need this in any case?

Reply:
I don't think we necessarily do. I'd prefer just keeping `push`/`replace` (see conversation [here](https://github.com/elastic/kibana/pull/176350#discussion_r1481511177)).

Reply:
Removing the `navigateToConversationsApp` in https://github.com/elastic/kibana/pull/176569/commits/20c6148bfde5a8213fa858d29c1263317fb6f633